Cartfile.resolved

```diff
@@ -1,5 +1,5 @@
 github "modocache/Guanaco" "26b6ceff4b8912abd0dfc3a9eed0a46527e040e3"
 github "Quick/Nimble" "v0.4.2"
 github "Quick/Quick" "155f858bce082373fc60d0c0a63415bf336c676b"
-github "antitypical/Result" "0.3"
+github "antitypical/Result" "0.3.2"
```

Comment:
Can you also update the `Cartfile` to specify that we need a release compatible with 0.3.2?